bad_code.py

```
@@ -0,0 +1,7 @@
+def add(a,b):
+   temp = "unused"
```

Ruff (0.14.4)

Local variable `temp` is assigned to but never used

Remove assignment to unused variable `temp`

(F841)